docs/advanced-guide/injecting-databases-drivers/page.md

```diff
@@ -712,4 +712,303 @@ func queryDataPoints(c *gofr.Context) (any, error) {
 	}
 	return queryResp.QueryRespCnts, nil
 }
-```
\ No newline at end of file
+```
+ScyllaDB
+
+
+GoFr supports injecting ScyllaDB to facilitate interaction with ScyllaDB REST APIs.Implementations adhering to the ScyllaDB interface can be registered with app.AddScyllaDB(),enabling applications to leverage ScyllaDB for time-series data management through gofr.Context
+```go
+type ScyllaDB interface {
+	// Query executes a CQL (Cassandra Query Language) query on the ScyllaDB cluster
+	// and stores the result in the provided destination variable `dest`
+	
+	Query(dest any, stmt string, values ...any) error
+	
+	//QueryWithCtx executes a CQL query with the provided context and stores the result in the `dest` variable
+	QueryWithCtx(ctx context.Context, dest any, stmt string, values ...any) error
+	
+	//Exec executes a CQL statement (e.g., INSERT, UPDATE, DELETE) on the ScyllaDB cluster without returning any result
+	Exec(stmt string, values ...any) error
+	
+	// ExecWithCtx executes a CQL statement with the provided context and without returning any result.
+	ExecWithCtx(ctx context.Context, stmt string, values ...any) error
+	
+	// ExecCAS performs a conditional (Compare and Set) operation on ScyllaDB, executing a CQL statement
+	ExecCAS(dest any, stmt string, values ...any) (bool, error)
+	
+	// NewBatch initializes a new batch operation with the specified name and batch type.
+	NewBatch(name string, batchType int) error
+	
+	// BatchQuery executes a batch query in the ScyllaDB cluster with the specified name, statement, and values.
+	BatchQuery(name, stmt string, values ...any) error
+	
+	// BatchQueryWithCtx executes a batch query with the provided context
+	BatchQueryWithCtx(ctx context.Context, name, stmt string, values ...any) error
+	
+	ExecuteBatchWithCtx(ctx context.Context, name string) error
+	
+	// HealthChecker defines the HealthChecker interface
+	HealthChecker
+}
+````
+
+
+Import the gofr's external driver for ScyllaDB:
+
+```go
+go get gofr.dev/pkg/gofr/datasource/scylladb
+```
```

Comment:
This one is about shell, not go


````suggestion
```shell
go get gofr.dev/pkg/gofr/datasource/scylladb
```
````